semgrep-core/Analyzing/Typing.ml

```diff
@@ -0,0 +1,35 @@
+open AST_generic
+open Lang
```

Comment:
I usually avoid more than 1 open. It can get more difficult to read the code not knowing where a constructor
could come from.